readme.md

```diff
@@ -175,6 +175,38 @@ Default: `process.cwd()`
 
 [Inform iTerm2](https://www.iterm2.com/documentation-escape-codes.html) of the current directory to help semantic history and enable [Cmd-clicking relative paths](https://coderwall.com/p/b7e82q/quickly-open-files-in-iterm-with-cmd-click).
 
+### iTerm.annotation(message, options?)
+
+#### message
+
+Type: `string`<br>
+
+The annotation contents.
+
+#### options
+
+Type: `object`
+
+##### length
+
+Type: `number`
+
+If present, annotate `length` number of cells. Defaults to the remainder of the line.
+
+##### x
+##### y
+
+Type: `number`
+
+Coordinates at which to begin annotation. If specified, `x`, `y`, *and* `length` are all required.
+
+##### hidden
+
+Type: `boolean`<br>
+Default: `false`
+
+If `true`, create a “hidden” annotation.
```

Comment:
Can you linkify "hidden annotation" to the iTerm docs about it. Not clear to a user what it is.

Reply:
hell, it's not even clear to _me_ what it is.  it just [says](https://iterm2.com/documentation-escape-codes.html) (sorry, no anchors) that the annotation is not immediately shown... but that's about it.

maybe we can remove the option until it's clear what is actually happening

Reply:
To me it seems that when it's on, the annotation is shown only on user gesture (hover or click) and not right away. Can't you just test and see?

Reply:
I did try that, but I couldn't make it show?  I'll give it another shot.